### PR TITLE
feat: Add Job page

### PR DESCRIPTION
**Pull request by gavin1ee**
add job list page and job add/update modal

### CODE REVIEW

File: src/Web/MASA.Scheduler.Web.Admin/Shared/Language.razor [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <ChildContent>
         <MList Class="py-0">
             <MListItemGroup ActiveClass="primary" Value="GlobalConfig.Language">
-                @foreach (var (language, map) in I18n.ToDictionary())
+                @foreach (var (language, map) in I18nCache.ToDictionary())
```
Comment (MayueCif) [inline src/Web/MASA.Scheduler.Web.Admin/Shared/Language.razor:13]: I18n.Languages

File: src/Web/MASA.Scheduler.Web.Admin.Server/Program.cs [hunk 2]
```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License. See LICENSE.txt in the project root for license information.
 
 using Masa.Scheduler.ApiGateways.Caller;
+using Masa.Stack.Components;
```
Comment (MayueCif) [inline src/Web/MASA.Scheduler.Web.Admin.Server/Program.cs:5]: global using

**Review by MayueCif — APPROVED** [review]

**Comment by MayueCif** [comment]
@doddgu

File: src/Contracts/MASA.Scheduler.Contracts.Server/Infrastructure/Enums/JobOriginTypes.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Scheduler.Contracts.Server.Infrastructure.Enums;
+
+public enum JobOriginTypes
```
Comment (doddgu) [inline src/Contracts/MASA.Scheduler.Contracts.Server/Infrastructure/Enums/JobOriginTypes.cs:6]: JobCreateTypes? and Manual, Api.

File: src/Contracts/MASA.Scheduler.Contracts.Server/Infrastructure/Utils/FluentValidationExtensions.cs [hunk 4]
```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Scheduler.Contracts.Server.Infrastructure.Utils;
+
+public static class FluentValidationExtensions
+{
+    public static IRuleBuilderOptions<T, string> Chinese<T>(this IRuleBuilder<T, string> ruleBuilder)
```
Comment (doddgu) [inline src/Contracts/MASA.Scheduler.Contracts.Server/Infrastructure/Utils/FluentValidationExtensions.cs:8]: Copy from mc?

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Contracts/MASA.Scheduler.Contracts.Server/Infrastructure/Utils/FluentValidationExtensions.cs (hunk 4 above)
Comment (gavin1ee) [reply src/Contracts/MASA.Scheduler.Contracts.Server/Infrastructure/Utils/FluentValidationExtensions.cs:8]: I will change it to use Masa.Utils.Extensions.Validations.FluentValidation when the library publish to nuget

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**